packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -89,58 +79,54 @@ class SmoothProductCardFound extends StatelessWidget {
             ),
             padding: const EdgeInsets.all(5.0),
             child: Row(
-              mainAxisAlignment: MainAxisAlignment.start,
-              crossAxisAlignment: CrossAxisAlignment.start,
               children: <Widget>[
-                Expanded(
-                  flex: 1,
-                  child: SmoothProductImage(
-                      product: product,
-                      width: screenSize.width * 0.20,
-                      height: screenSize.width * 0.20),
-                ),
-                const SizedBox(
-                  width: 8.0,
+                SmoothProductImage(
+                  product: product,
+                  width: screenSize.width * 0.20,
+                  height: screenSize.width * 0.20,
                 ),
+                const SizedBox(width: 5),
                 Expanded(
-                  flex: 2,
-                  child: Column(
-                    mainAxisSize: MainAxisSize.max,
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: <Widget>[
-                      SizedBox(
-                        width: screenSize.width * 0.65,
-                        child: Column(
+                  child: Container(
+                    height: screenSize.width * 0.2,
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          product.productName ?? '???',
+                          overflow: TextOverflow.ellipsis,
+                          style: const TextStyle(
```

Comment:
can the style be reused from the theme?

Reply:
Better?

Reply:
definitely, if you still want to alter some style in the textstyle from theme you can do it with `.apply` function